src/Illuminate/Database/Schema/Builder.php

```diff
@@ -325,6 +325,27 @@ public function dropColumns($table, $columns)
         });
     }
 
+    /**
+     * Drop index from a table schema only if exists.
+     *
+     * @param  string  $table
+     * @param  string  $index
+     * @return void
+     */
+    public function dropIndexIfExists($table, $index)
+    {
+        $sm = $this->getConnection()->getDoctrineSchemaManager();
+        $indexes = $sm->listTableIndexes($table);
+        
+        if (array_key_exists($index, $indexes)) { 
+            $this->table($table, function (Blueprint $blueprint) use($index) {
```

Comment:
```suggestion
            $this->table($table, function (Blueprint $blueprint) use ($index) {
```

StyleCI is [failing here](https://github.styleci.io/analyses/YOZ3d0?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) ⬆️

Reply:
@devajmeireles good idea. i modified my code to also accept array of indexes as well.